lib/formats/sh.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 PROGRESS_CURR=0
-PROGRESS_TOTAL=                            
+PROGRESS_TOTAL=
```

Comment:
We need these spaces here. The actual PROGRESS_TOTAL is written in the "blank space" by the hacky seek code. Please keep them.